README.md

```diff
@@ -17,6 +17,14 @@ especially if you are using zopflipng for compression.  Intermediate products
 (compressed image files, for example) will be put into a build subdirectory; the
 font will be at the top level.
 
+## Using NotoColorEmoji
+
+NotoColorEmoji uses the CBDT/CBLC color font format, which is supported by Android
+and ChromeOS, but neither Windows nor MacOS.  No Browser on MacOS supports it, but
```

Comment:
Thanks Sascha, this slipped by us.  Behdad had pointed out that it was supported in the OS and not just Edge, but I failed to update the README.  It's fixed now.